tests/ltp/Makefile

```diff
@@ -29,6 +29,10 @@ $(ROOT_FS): $(ALPINE_TAR) buildenv.sh
 	$(ESCALATE_CMD) mount -t ext4 -o loop "$@" $(MOUNTPOINT)
 	$(ESCALATE_CMD) tar -C $(MOUNTPOINT) -xvf $(ALPINE_TAR)
 	$(ESCALATE_CMD) cp /etc/resolv.conf $(MOUNTPOINT)/etc/resolv.conf
+	git submodule init $(SGXLKL_ROOT)/ltp
+	git submodule update --remote $(SGXLKL_ROOT)/ltp
+	$(ESCALATE_CMD) mkdir $(MOUNTPOINT)/ltp
+	$(ESCALATE_CMD) cp -R $(SGXLKL_ROOT)/ltp/* $(MOUNTPOINT)/ltp/
```

Comment:
why isnt this just a symbolic link?

Reply:
@paulcallen here we are copying into image through mounted folder. I am not sure if image will work with symlink and also we umount after copying ltp/ folder into image. LTP tests run from image directly.

Reply:
We pass a fresh image file for running each LTP test and using symlink might not work for this. We will keep as is and it is already super fast. If we think we need such optimization we can submit another PR with necessary code changes with symlink usage and good testing to make sure nothing wil be broken. Merging as is.